src/Nethermind/Nethermind.Blockchain/BlockTree.cs

```diff
@@ -827,6 +827,11 @@ public void DeleteInvalidBlock(Block invalidBlock)
             }
         }
 
+        public void Flush()
```

Comment:
BlockTree.Flush? let us not pllute the API with such an internal thing - can we recognize to flush on BlockProcessed for example?